### PR TITLE
CAP-38: Add protocol version

### PR DESCRIPTION
**Pull request by MT-Cash**
See https://developers.stellar.org/docs/glossary/liquidity-pool/

### CODE REVIEW

**Comment by jonjove** [comment]
Thank you for adding this. Typically we only make this update once the CAP is live on the production network (in addition, we would change the status to "Final"). Assuming the vote goes through, we'll merge this in a few weeks.

**Comment by MT-Cash** [comment]
Oh, sorry, I wasn't aware of that; I thought it was simply forgotten :sweat_smile:

Feel free to close the PR; I've updated it in case you want to use it when it actually is live :-)

**Review by jonjove — APPROVED** [review]

File: core/cap-0038.md [hunk 1]
```diff
@@ -7,10 +7,10 @@ Working Group:
     Owner: Jonathan Jove <@jonjove>
     Authors: Jonathan Jove <@jonjove>, Siddharth Suresh <@sisuresh>
     Consulted: OrbitLens <@orbitLens>, Nikhil Saraf <@nikhilsaraf>, Tamir Sen <@tamirms>, Phil Meng <@phil-stellar>, Leigh McCulloch <@leighmcculloch>, Nicolas Barry <@monsieurnicolas>
-Status: Accepted
+Status: Final
```
Comment (leighmcculloch) [inline core/cap-0038.md:10]: @jonjove The status also needs updating in the CAP table in the readme if we're changing its status here.

Comment (jonjove) [reply core/cap-0038.md:10]: @leighmcculloch fixed that and several other status issues in #1085

**Pull request merged**